shared/client/client.go

```diff
@@ -0,0 +1,56 @@
+package phoneValidation
+
+import (
+	"io"
+	"kargo-back/shared/environment"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/gojektech/heimdall"
+	"github.com/gojektech/heimdall/httpclient"
+)
+
+var (
+	httpClientTimeout           = time.Duration(environment.GetInt64("HTTP_CLIENT_TIMEOUT", 5)) * time.Second
```

Comment:
Lower that default timeout, 5 seconds is way too much

Reply:
Fun fact: the recommended is 10 because of low level telecom protocols, anyways this is an env var we change it easily